sematic/examples/mnist/pytorch/pipeline.py

```diff
@@ -61,13 +62,17 @@ class PipelineConfig:
 # GPUs will vary depending on your Kubernetes setup.
 GPU_RESOURCE_REQS = ResourceRequirements(
     kubernetes=KubernetesResourceRequirements(
-        node_selector={"node.kubernetes.io/instance-type": "g4dn.xlarge"},
-        requests={"cpu": "2", "memory": "4Gi"},
+        node_selector={
+            "cloud.google.com/gke-nodepool": "gpu-t4-1",
+            "cloud.google.com/gke-accelerator": "nvidia-tesla-t4",
+        },
+        requests={"cpu": "2", "memory": "4Gi", "nvidia.com/gpu": "1"},
```

Comment:
This reflects a change in the way to request a GPU from our old AWS cluster to the GCP one.